### PR TITLE
Update experiment-rollout.js

### PR DESCRIPTION
**Pull request by Lordcobcob**
Added An S to The Moderator Programs Alumni it was annoying me ok

### CODE REVIEW

File: experiment-rollout.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const data = {
             },
             {
                 status: 0,
-                name: 'Moderator Program Alumni'
+                name: 'Moderator Programs Alumni'
```
Comment (Tolga1452) [inline experiment-rollout.js:89]:
```suggestion
                name: 'Moderator Programs Alumnis'
```
:)

Comment (Tolga1452) [reply experiment-rollout.js:89]: also and `'HypeSquad Event owners'`

**Review by Lordcobcob — COMMENTED** [review]
help its not letting me change it it errors it every time

**Review by Lordcobcob — COMMENTED** [review]
help its not letting me change it it errors it every time

**Pull request closed without merging**